case-study-savas-labs.mdx

```diff
@@ -0,0 +1,86 @@
+
+## The task ahead
+
+Savas Labs was approached by the [Environmental Integrity Project](https://environmentalintegrity.org/) (EIP) to do a complete teardown and replacement of an existing Angular 1.0 application backed by a Postgres database. The purpose of the site is to show complicated environmental data (CO2e, economic effects, cancer risk, projected deaths, water crossings) of hundreds of facilities and pipelines throughout the United States, and display those in a way that is digestible to the user; be they an environmental scientist, legal professional, or curious citizen.
+
+## Challenges
+
+The existing [Oil & Gas Watch project](https://oilandgaswatch.org/) platform presented several challenges:
+
+- **Avoiding database management:** A The Savas Labs team wanted to avoid the complexities of managing a database infrastructure themselves.
+- **API accessibility:** A They needed a solution that could easily provide an API layer over the existing Postgres database without the need for extensive backend development.
+- **Data migration:** A Importing an existing Postgres database into a new system was a critical requirement.
+- **Geospatial data management:** A Savas Labs needed a solution to efficiently handle geospatial data with robust querying capabilities and integration with mapping tools. In addition, handling [Postgres branching](https://xata.io/branching) for different project environments posed unique challenges, which required Xata’s support.
+- **Collaborative visibility:** A The team required a platform that enabled visibility into data for better collaboration and project management.
+
+## Solution
+
+After evaluating several Database-as-a-Service (DBaaS) solutions, Savas Labs chose Xata for its simplicity and effectiveness in meeting their specific needs.
+
+![Building with Xata](/images/savas-xata-table.png)
+
+**Key factors for choosing Xata:**
+
+- **Simplicity and speed:** Xata's ability to load data quickly and provide an immediate API endpoint was a significant advantage. The team could focus on data presentation rather than management , effectively handling tasks such as [Postgres schema migrations](https://xata.io/schema-migrations) without extensive manual intervention.
+- **Personal touch:** Xata's proactive support in importing an existing Postgres database, and their involvement in the onboarding process, stood out as major differentiators.
+- **UI experience:** The user-friendly interface and collaborative features of Xata's tables facilitated smooth teamwork, making it easy for project managers to monitor progress without constantly pinging developers.
+- **Engender Client Control:** The EIP are experts in environmental data, and Savas wanted them to do what they do best. By allowing EIP full control over all their location records, they could focus on getting the data right, and Savas could focus on how it would display.
```

Comment:
Is Engender Client Control the right description? I'm not sure what this is and quick googling didn't produce much. Just want to make sure it's not a typo. I haven't been following the Savas changes / suggestions closely

Reply:
Its a bullet point that Lemon added in.